mon-pix/app/mixins/progression-tracker.js

```diff
@@ -17,9 +18,9 @@ export default Mixin.create({
 
   init() {
     this._super(...arguments);
-    this.stepsAhead = this.steps.map(
+    set(this, 'stepsAhead', this.steps.map(
       (step) => EmberObject.create({ name: step, status: STEPS.TO_COMPLETE }),
-    );
+    ));
```

Comment:
**question :** Pourquoi ce changement ? Je pensais qu'on utilisait plus les set/get ?

Reply:
Bonne question, ça a piqué ma curiosité. J'avais lancé un autofix du linter au moment du bump et j'ai donc splitté le commit en intention. 

L'intention du fix dans ce fichier était la suivante : 
> error  Use `set(this, 'propertyName', 'value')` instead of assignment for untracked properties that are used as computed property dependencies (or convert to using tracked properties)  ember/no-assignment-of-untracked-properties-used-in-tracking-contexts

Je vais voir pour passer en tracked afin de supprimer le set sinon je laisse comme ça

Reply:
Après discussion avec @sbedeau , on décide d'avancer en laissant tel quel pour le moment, en espérant que d'ici 6 mois (18 si on est honnêtes avec nous-mêmes) les mixins aient disparues des Pix fronts.